src/main/java/ChatbotRan/Task.java

```diff
@@ -0,0 +1,37 @@
+package ChatbotRan;
+
+public abstract class Task {
+    protected String contents;
+    protected boolean completed;
+
+    public Task(String contents) {
+        this.contents = contents;
+        this.completed = false;
```

Comment:
While completed might be clear that it is a boolean value, convention is to name such variables as isCompleted or hasCompleted
```suggestion
        this.isCompleted = false;
```